src/pages/experiments/[experimentId]/plots-and-tables/embedding-continuous/index.jsx

```diff
@@ -58,7 +58,7 @@ const ContinuousEmbeddingPage = ({ experimentId }) => {
   const [searchedGene, setSearchedGene] = useState();
 
   useEffect(() => {
-    if (config?.shownGene !== 'notSelected' && config && !searchedGene) {
+    if (config?.shownGene !== null && config && !searchedGene) {
       // if there is a saved gene in the config in the initial loading of the plot
```

Comment:
this comment does not clarify what's happening to me. "If there is a saved gene in the config" I assume refers to `config?.shownGene` but then what about the `!searchedGene`? Could you reword this to make it more clear?

also I'd probably reorder the if so the `?` is not needed: `config && config.shownGene !== null && !searchedGene` but I leave it to your decision

Reply:
We don't need to check `config` at all, just doing `config?.shownGene !== null` handles config being `null` too.

About the comment, I took a brief look at what it is doing and updated the comment, but based on what I saw this could benefit from a more integral refactoring probably to make `searchedGene`, `shownGene` and the expressions loading a little bit more cohesive. This PR is about fixing the bugs though so I am not sure if perhaps I shoudl create a ticket for that.

Reply:
I just meant to improve the comment which you did so now it's clear, thanks; and yeah the `config` can be removed alltogether.